free-programming-books-zh.md

```diff
@@ -640,12 +640,17 @@
 ### Python
 
 * [Django 1.11.6 中文文档](https://www.yiyibooks.cn/xx/Django_1.11.6/index.html)
+* [Django 2.2.1 中文文档](http://www.osgeo.cn/django/) - (这个很新，也很全，Online)
```

Comment:
I think we should remove describe. we just need author and type after the line.
```suggestion
* [Django 2.2.1 中文文档](http://www.osgeo.cn/django/)
```
so as other resource.